teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Kindly use the colon notation to assign the `name` and  `parent_permission` parameters to avoid ArgumentError as you already use the colon notation in the parent class(Person), also stick with the order of parameters in the parent class as the `name` parameter comes after `age`, so it will be like:
```ruby
super(age, name: name, parent_permission: parent_permission)
```

Reply:
Thanks